doc/bootstrap.rst

```diff
@@ -389,6 +388,38 @@ Example of a Hosts Pool deployment configuration file
       private_address: "10.0.0.11"
       public_address: "10.129.1.11"
 
+
+Exporting and loading an interactive configuration file
+------------------------------------------
+
+When deploying, the final configuration of the bootstrapping is automatically exported to a file. The name of the 
+file is the deployment id, which is a timestamp of current year to second. You can create a custom deployment id 
+using ''-n'' option :
+
+.. parsed-literal::
+
+    ./yorc bootstrap -n a_deploy_name 
+
+If you specify an already existing name (an input config file of the same name this already exists), an unique name will
+be created, of the form ''nameN'', where N is an integer, generated incrementally. 
+
+You can then load a config file using the "-v" option :
+
+.. parsed-literal::
+
+    ./yorc bootstrap -v an_existing_deploy_name

```

Comment:
not an_existing_deploy_name, should be:
./yorc bootstrap -v <path to file containing input values>

Reply:
you are right !